<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Система персонализированых индивидуальных ключей (СПИК)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="source.qrc">
    <normaloff>:/mainapp/sources/appico.ico</normaloff>:/mainapp/sources/appico.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="profileTab">
     <attribute name="icon">
      <iconset resource="source.qrc">
       <normaloff>:/mainapp/sources/profile.ico</normaloff>:/mainapp/sources/profile.ico</iconset>
     </attribute>
     <attribute name="title">
      <string>Профиль</string>
     </attribute>
     <widget class="QLineEdit" name="idField">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>371</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Ошибка ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_id">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>9</y>
        <width>371</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ваш ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_login">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ваш Логин</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="loginField">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>281</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ваш логин</string>
      </property>
     </widget>
     <widget class="QListWidget" name="contactList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>371</width>
        <height>301</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>username111</string>
       </property>
       <property name="icon">
        <iconset resource="source.qrc">
         <normaloff>:/mainapp/sources/contacts.ico</normaloff>:/mainapp/sources/contacts.ico</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>paswedfaxsz123</string>
       </property>
       <property name="icon">
        <iconset resource="source.qrc">
         <normaloff>:/mainapp/sources/contacts.ico</normaloff>:/mainapp/sources/contacts.ico</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>qwerety123</string>
       </property>
       <property name="icon">
        <iconset resource="source.qrc">
         <normaloff>:/mainapp/sources/contacts.ico</normaloff>:/mainapp/sources/contacts.ico</iconset>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_contacts">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Контакты&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_keys">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>80</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Ключи&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QListWidget" name="keysList">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>110</y>
        <width>361</width>
        <height>391</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Публичный ключ аккаунта</string>
       </property>
       <property name="icon">
        <iconset resource="source.qrc">
         <normaloff>:/mainapp/sources/keys.ico</normaloff>:/mainapp/sources/keys.ico</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Приватный ключ аккаунта</string>
       </property>
       <property name="icon">
        <iconset resource="source.qrc">
         <normaloff>:/mainapp/sources/keys.ico</normaloff>:/mainapp/sources/keys.ico</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ключ чата (123)</string>
       </property>
       <property name="icon">
        <iconset resource="source.qrc">
         <normaloff>:/mainapp/sources/keys.ico</normaloff>:/mainapp/sources/keys.ico</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ключ чата (Название чата)</string>
       </property>
       <property name="icon">
        <iconset resource="source.qrc">
         <normaloff>:/mainapp/sources/keys.ico</normaloff>:/mainapp/sources/keys.ico</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ключ диалога (username111)</string>
       </property>
       <property name="icon">
        <iconset resource="source.qrc">
         <normaloff>:/mainapp/sources/keys.ico</normaloff>:/mainapp/sources/keys.ico</iconset>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="setLogin">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Изменить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="updatePas">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>371</width>
        <height>24</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Обновить пароль</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addContact">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeContact">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>490</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="i2pAddressField">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>31</y>
        <width>371</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Ошибка ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_i2padr">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>371</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ваш i2p-адрес</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="messagesTab">
     <attribute name="icon">
      <iconset resource="source.qrc">
       <normaloff>:/mainapp/sources/messages.ico</normaloff>:/mainapp/sources/messages.ico</iconset>
     </attribute>
     <attribute name="title">
      <string>Сообщения</string>
     </attribute>
     <widget class="QLabel" name="label_messages">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Сообщения&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QListWidget" name="messagesField">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>300</width>
        <height>491</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>123</string>
       </property>
       <property name="icon">
        <iconset resource="source.qrc">
         <normaloff>:/mainapp/sources/chat.ico</normaloff>:/mainapp/sources/chat.ico</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Название чата</string>
       </property>
       <property name="icon">
        <iconset resource="source.qrc">
         <normaloff>:/mainapp/sources/chat.ico</normaloff>:/mainapp/sources/chat.ico</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>username111</string>
       </property>
       <property name="icon">
        <iconset resource="source.qrc">
         <normaloff>:/mainapp/sources/dialog.ico</normaloff>:/mainapp/sources/dialog.ico</iconset>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_dialog">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>0</y>
        <width>471</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Диалог&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="dialogField">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>471</width>
        <height>461</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:7pt; color:#8a8a8a;&quot;&gt;Генерация и передача ключей прошла успешно, приятного общения!&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:7pt; color:#8a8a8a;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700; color:#aa007f;&quot;&gt;[username111]&lt;/span&gt;: Привет&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700; color:#55007f;&quot;&gt;[Вы]&lt;/span&gt;: Даров&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="messageArea">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>490</y>
        <width>391</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sendButton">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>490</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Отправить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="storageTab">
     <attribute name="icon">
      <iconset resource="source.qrc">
       <normaloff>:/mainapp/sources/storage.ico</normaloff>:/mainapp/sources/storage.ico</iconset>
     </attribute>
     <attribute name="title">
      <string>Хранилище</string>
     </attribute>
     <widget class="QLabel" name="inDevTxt">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>521</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:700; color:#8a8a8a;&quot;&gt;В разработке&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Программа</string>
    </property>
    <widget class="QMenu" name="menuSettings">
     <property name="title">
      <string>Настройки</string>
     </property>
     <addaction name="separator"/>
     <addaction name="generalSettingsAction"/>
     <addaction name="settingsI2pAction"/>
     <addaction name="settingsFreenetAction"/>
    </widget>
    <widget class="QMenu" name="logoutMenu">
     <property name="title">
      <string>Выйти из учётной записи</string>
     </property>
     <addaction name="logoutNow"/>
     <addaction name="logoutLater"/>
    </widget>
    <addaction name="menuSettings"/>
    <addaction name="logoutMenu"/>
    <addaction name="separator"/>
    <addaction name="creditsAction"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Язык (Language)</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="ruLocaleAction"/>
    <addaction name="enLocaleAction"/>
    <addaction name="uaLocaleAction"/>
   </widget>
   <widget class="QMenu" name="menuAccount">
    <property name="title">
     <string>Аккаунт</string>
    </property>
    <addaction name="saveAccAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuAccount"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="generalSettingsAction">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/mainapp/sources/profile.ico</normaloff>:/mainapp/sources/profile.ico</iconset>
   </property>
   <property name="text">
    <string>Общие настройки</string>
   </property>
  </action>
  <action name="settingsI2pAction">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/mainapp/sources/i2p.png</normaloff>:/mainapp/sources/i2p.png</iconset>
   </property>
   <property name="text">
    <string>Настройки i2p</string>
   </property>
  </action>
  <action name="settingsFreenetAction">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/mainapp/sources/freenet.png</normaloff>:/mainapp/sources/freenet.png</iconset>
   </property>
   <property name="text">
    <string>Настройки freenet</string>
   </property>
  </action>
  <action name="logoutNow">
   <property name="text">
    <string>Сейчас</string>
   </property>
  </action>
  <action name="logoutLater">
   <property name="text">
    <string>Задать время автовыхода</string>
   </property>
  </action>
  <action name="ruLocaleAction">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/locales/langs/rus.png</normaloff>:/locales/langs/rus.png</iconset>
   </property>
   <property name="text">
    <string>Русский (Russian)</string>
   </property>
  </action>
  <action name="enLocaleAction">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/locales/langs/eng.jpg</normaloff>:/locales/langs/eng.jpg</iconset>
   </property>
   <property name="text">
    <string>English (English)</string>
   </property>
  </action>
  <action name="uaLocaleAction">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/locales/langs/ua.jpg</normaloff>:/locales/langs/ua.jpg</iconset>
   </property>
   <property name="text">
    <string>(β) Український (Ukrainian)</string>
   </property>
  </action>
  <action name="creditsAction">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Об авторе</string>
   </property>
  </action>
  <action name="saveAccAction">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
